app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<% @contestants.each do |contestant| %>
+  <h2><%= contestant.name %></h2>
+  <p>Projects: <%= contestant.projects.map(&:name).join(", ") %></p>
```

Comment:
Before I speak more to this line of code here, I'll wait on your response about why you felt you needed to use `include`.